create_rental.rb

```diff
@@ -0,0 +1,26 @@
+class CreateRental
+  def initialize(state)
+    @state = state
+    @book_list = @state[:book_list]
+    @people_list = @state[:people_list]
+    @rental_list = @state[:rental_list]
+  end
+
+  def create_rental
+    p 'Select a book from the following list by number'
+    @book_list.each_with_index do |book, index|
+      p "#{index}) Title: '#{book.title}', Author: #{book.author}"
+    end
+    book = @book_list[gets.chomp.to_i]
+    p 'Select a person from the following list by number (not id)'
+    @people_list.each_with_index do |person, index|
+      p "#{index}) [#{person.class.name}] Name: '#{person.name}', ID: #{person.id}, Age: #{person.age}"
+    end
+    person = @people_list[gets.chomp.to_i]
+    print 'Date: '
+    date = gets.chomp
```

Comment:
- [ ] In keeping with the single-responsibility rule, kindly ensure these requests are handled by another method/class.

Reply:
Hi @TedLivist, Thank you for the helpful feedback 🙏 . We have implemented and pushed all the suggested changes.